docs/source/theory/fktables.rst

```diff
@@ -0,0 +1,130 @@
+.. _fktables:
+
+============================================================
+Fast Interface (FK tables)
+============================================================
+
+.. raw:: latex
+
+   \maketitle
+
+.. raw:: latex
+
+   \tableofcontents
+
```

Comment:
move the motivation up (so move the paragraphs below)

Reply:
Do you mean the part where I say "The compuation of the above equations..."?

Reply:
"The direct calculation" and "to evaluate" - or just express in your own words why we do it; just to give an external reader some sort of guideline ...